src/components/AdminPage/AdminPage.js

```diff
@@ -4,19 +4,20 @@ import MusicianCard from '../MusicianCard/MusicianCard';
 import { useQuery } from '@apollo/client';
 import { GET_SUBMISSIONS } from '../../queries';
 
-const AdminPage = () => {
+const AdminPage = ({ user }) => {
   const [musicians, setMusicians] = useState([]);
+
   const { loading, error, data } = useQuery(GET_SUBMISSIONS);
 
+  const renderCards = (data) => {
+    return data.getSubmissions.map(submission => {
+      return <MusicianCard key={submission.id} submission={submission} />
+    });
+  }
+
```

Comment:
Originally moved this block into the 'if' statement because I was getting an error if the array was empty. Maybe we could add some additional logic to check the length of the array first before rendering the cards and display a message if the array is empty???

Reply:
Resolved this our video call. Many thanks!